src/main/java/net/barrage/school/java/ecatalog/app/merchantService/MerchantRepostiory.java

```diff
@@ -0,0 +1,13 @@
+package net.barrage.school.java.ecatalog.app.merchantService;
```

Comment:
I'd better not to use camel case for package names. There is convention for it - https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html#:~:text=Naming%20Conventions,a%20programmer%20at%20example.com%20.

Reply:
Also why not to create following packages. It will be more like java-style.
- `app.product`
- `app.product.source`
- `app.merchnat`